web/v1/snowflake/CHANGELOG

```diff
@@ -0,0 +1,5 @@
+Version 1.0.0 (2021-01-29)
+--------------------------
+Fix bug in credentials handling for pr_check and e2e scripts (#35)
```

Comment:
This line will go into new top-level changelog (see bigquery 1.0.1 release). 

If this PR is ready first, then we can leave it as is and I'll change it in the other PR. :)

Reply:
Other PR is going after this release, so this comment need not apply.